WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerCollectionViewCell.swift

```diff
@@ -0,0 +1,125 @@
+
+class LayoutPickerCollectionViewCell: UICollectionViewCell {
+
+    static var cellReuseIdentifier: String {
+         return "LayoutPickerCollectionViewCell"
+     }
+
+    static var nib: UINib {
+        return UINib(nibName: "LayoutPickerCollectionViewCell", bundle: Bundle.main)
+    }
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var checkmarkBackground: UIView!
+    @IBOutlet weak var checkmarkImageView: UIImageView! {
+        didSet {
+            if #available(iOS 13.0, *) {
+                checkmarkImageView.image = UIImage(systemName: "checkmark.circle.fill")
+            } else {
+                checkmarkImageView.image = UIImage.gridicon(.checkmarkCircle)
+            }
+
+            checkmarkImageView.tintColor = accentColor
+        }
+    }
+
+    var layout: GutenbergLayout? = nil {
+        didSet {
+            setImage(layout?.preview)
+        }
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        imageView.cancelImageDownload()
+    }
+
+    var accentColor: UIColor {
+        if #available(iOS 13.0, *) {
+            return UIColor { (traitCollection: UITraitCollection) -> UIColor in
+                if traitCollection.userInterfaceStyle == .dark {
+                    return UIColor.muriel(color: .accent, .shade40)
+                } else {
+                    return UIColor.muriel(color: .accent, .shade50)
+                }
+            }
+        } else {
+            return UIColor.muriel(color: .accent, .shade50)
+        }
+    }
+
+    var borderColor: UIColor {
+        return UIColor.black.withAlphaComponent(0.08)
+    }
+
+    var borderWith: CGFloat = 0.5
+
+    override var isSelected: Bool {
+        didSet {
+            styleSelectedBorderColor()
+            imageView.layer.borderWidth = isSelected ? 2 : borderWith
+            checkmarkImageView.isHidden = !isSelected
+            checkmarkBackground.isHidden = !isSelected
+        }
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        styleSelectedBorderColor()
+        imageView.layer.borderWidth = borderWith
+
+        if #available(iOS 13.0, *) {
+             styleShadow()
+         } else {
+             addShadow()
+         }
+    }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if #available(iOS 13.0, *) {
+            if traitCollection.hasDifferentColorAppearance(comparedTo: previousTraitCollection) {
+                styleSelectedBorderColor()
+                styleShadow()
+            }
+        }
+    }
+
+    @available(iOS 13.0, *)
+    func styleShadow() {
+        if traitCollection.userInterfaceStyle == .dark {
+            removeShadow()
+        } else {
+            addShadow()
+        }
+    }
+
+    func addShadow() {
+        layer.shadowColor = UIColor.black.cgColor
+
+        let scale = UIScreen.main.scale
```

Comment:
I'm dividing by the scale on these values to get closer to what the design specs have however this might be temporary as we continue to refine the shadow in design review. I updated this in the [category bar PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/14631/files#diff-a54f62ed5a5714d192331c0df3844353R103-R105) to remove this scale and use the same point value.